packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -599,12 +602,25 @@ export interface TakeLoanMetadata {
   to?: string
 }
 
-export interface PaybackLoanMetadata {
+export interface PaybackLoanMetadataV1 {
```

Comment:
hmm... wondering do we still need this??

Reply:
Your mean to keep the old interface as it is and change the method signature like this `paybackLoan (metadata: PaybackLoanMetadata | PaybackLoanMetadataV2, utxos: UTXO[] = [])`, right ?

Reply:
i was thinking to combine them but yours is right